hail/src/main/scala/is/hail/linalg/LAPACK.scala

```diff
@@ -0,0 +1,87 @@
+package is.hail.linalg
+
+import java.lang.reflect.Method
+
+import com.sun.jna.{FunctionMapper, Library, Native, NativeLibrary, Pointer}
+import com.sun.jna.ptr.{DoubleByReference, IntByReference}
+
+import scala.util.{Failure, Success, Try}
+import is.hail.utils._
+
+class UnderscoreFunctionMapper extends FunctionMapper {
+  override def getFunctionName(library: NativeLibrary, method: Method): String = {
+    method.getName() + "_"
+  }
+}
+
+object TryAll {
+  def apply[K](f: => K): Try[K] =
+    try {
+      Success(f)
+    } catch {
+      case e: Throwable => Failure(e)
+    }
+}
```

Comment:
I'm pretty sure this is literally what the `Try(...)` is?

Reply:
The builtin `Try` only catches "non fatal" errors (I think this means classes that end in `Exception`, but not ones that end in `Error`). Failing to link is a fatal error, so regular `Try` doesn't catch it.

Reply:
🙄